<!--This file created 4:37 PM  12/11/97 by Claris Home Page version 2.0-->
<HTML>
<HEAD>
   <TITLE>RECOMB 98 - Further Information</TITLE>
   <META NAME=GENERATOR CONTENT="Claris Home Page 2.0">
   <X-SAS-WINDOW TOP=0 BOTTOM=659 LEFT=0 RIGHT=1020>
</HEAD>
<BODY BGCOLOR="#FBF8BC">

<H1>ACCEPTED PAPERS</H1>

<P>&nbsp;</P>

<PRE>PAPER 001  
P. Agarwal and V. Bafna 
Detecting non-adjoining correlations within signals in DNA.
&nbsp;
PAPER 002 
A. Ben-Dor, B. Chor, D. Graur, R. Ophir, D. Pelleg  
From four-taxon trees to phylogenies: The case of mammalian evolution.
&nbsp;
PAPER 003
G. Benson  
An Algorithm for Finding Tandem Repeats of Unspecified Pattern 
&nbsp;
PAPER 004
B. Berger, T.Leighton 
Protein Folding in the Hydrophobic-Hydrophilic (HP) Model is NP-Complete.
&nbsp;
PAPER 005
M. Bonet, M.Steel, T.Warnow, S.Yooseph
Better Methods for Solving Parsimony and Compatiblity
&nbsp;
PAPER 006
C. C. Chen, J.P. Singh, R.B. Altman 
The Hierarchical Organization of Molecular Structure Computations
&nbsp;
PAPER 007
P. Crescenzi, D. Goldman, C. Papadimitriou, A. Piccolboni, M. Yannakakis 
On the Complexity of Protein Folding.
&nbsp;
PAPER 008
D. Fasulo, T. Jiang, R. M. Karp, N. Sharma
Constructing Maps Using the Span and Inclusion Relations.
&nbsp;
PAPER 009
A. V. Finkelstein, A. Ya. Badretdinov 
How Fast a Protein Chain Can Fold to Its Most Stable Structure?
&nbsp;
PAPER 010
R. M. Fye,  C.J. Benham
A Formally Exact Method to Numerically Analyze Local Denaturation in 
Superhelical DNA
&nbsp;
PAPER 011
I. Gelfand, A. Kister, C. Kulikowski, O. Stoyanov    
Algorithmic Determination of Core Positions in the VL and VH Domains of 
Immunoglobulin Molecules
&nbsp;
PAPER 012
R. A. Goldstein, J. M.Koshi, D. P.Mindell  
Beyond mutation matrices: Physical-Chemistry based evolutionary models.
&nbsp;
PAPER 013
W. N. Grundy  
Family-based Homology Detection via Pairwise Sequence Comparison.
&nbsp;
PAPER 014     
I. Holmes and R. Durbin 
Dynamic Programmimg Alignment Accuracy.
&nbsp;
PAPER 015
T. Hwa, M. Lassig 
Optimal Detection of Sequence Similarity by Local Alignment.
&nbsp;
PAPER 016
R.M.Karp, R.Shamir  
Algorithms for Optical Mapping.
&nbsp;
PAPER 017
P. E. Kearney  
Ordinal Beats Additive
&nbsp;
PAPER 018
E. Koonin, R. Tatusov, M.Y. Galperin and M.N. Rozanov  
Genome analysis using clusters of orthologous groups (COGs)
&nbsp;
&nbsp;
PAPER 019
J. K.Lee, V. Dancik, M. S. Waterman
Estimation for restriction sites observed by optical mapping using 
reversible-jump markov chain monte carlo
&nbsp;
PAPER 020
H. Lenhof, K. Reinert, M. Vingron 
A Polyhedral Approach to RNA Sequence Structure Alignment.
&nbsp;
PAPER 021
A. M. Lesk 
Assessment of Ab Initio Protein Structure Prediction.
&nbsp;
PAPER 022   
R. Liu, T. W. Blackwell and D. J. States 
A structure based similarity measure for nucleic acid sequence comparison.
&nbsp;
PAPER 023
B. Ma, M. Li, L. Zhang 
On Reconstructing Species Trees From Gene Trees In Term of Duplications 
&nbsp;
PAPER 024
L. Parida, B. Mishra  
Partitioning K Clones: Hardness Results and a Practical Solution to 
the K-Popultaions Problem
&nbsp;
PAPER 025
D. G.Politte, D. R. Maffitt, D. J.States 
Estimaiton of Allele Frequencies From Color-Multiplexed Electropherograms
&nbsp;
PAPER 026
M. Regnier 
A Unified Approach to Word Statistics
&nbsp;
PAPER 027 
B. A.Reva, A. V.Finkelstein, J. Skolnick      
A self-consistent field optimization approach to build energetically and 
geometrically correct lattice models of proteins 
&nbsp;
PAPER 028
I. Rigoutsos,A. Floratos  
Motif Discovery in Biological Sequences Without Alignment or Enumeration
&nbsp;
PAPER 029 
E. Rocke, M. Tompa
An Algorithm for Finding Novel Gapped Motifs in DNA sequences
&nbsp;
PAPER 030  
M. Sagot, E. W. Myers
Identifying satellites in nucleic acid sequences
&nbsp;
PAPER 031
D. Sankoff, M. Blanchette 
Multiple genome rearrangement
&nbsp;
PAPER 032
F. Sun
Modeling DNA shuffling
&nbsp;
PAPER 033
S. R. Sunyaev, I. V. Rodchenkov
Analysis of the Position Dependent Amino Acid Probabilities and its
Application
to the Search for Remote Homologues.
&nbsp;
PAPER 034
A. Vologodskii
Maxwell demon and topology simplification by type II topoisomerases.
&nbsp;
PAPER 035
W. L.Walker, D. S. Goodsell,E. M.Landaw    
The Theoretical Limits of DNA sequence discrimination of Polyamides
&nbsp;
PAPER 036
T. D. Wu, S. C. Schmidler, T. Hastie, D. L. Brutlag
Regression Analysis of Multiple Protein Structures
&nbsp;
PAPER 037
Y. Xu, D. Xu, E. C. Uberbacher 
A New Method for modeling and solving the protein fold recognition problem
&nbsp;
PAPER 038
Z. Zhang, P. Berman, W. Miller 
Alignments Without Low-Scoring Regions 
&nbsp;
&nbsp;
&nbsp;</PRE>

<P>
<HR>
</P>

<H5><I><A HREF="recomb98.html">Back to RECOMB 98 Home
page</A></I></H5>

<H5><I><A HREF="https://recomb-conference.github.io/recomb1997">Link to RECOMB
97</A></I></H5>

<P>&nbsp;</P>

<P>&nbsp;</P>

</BODY>
</HTML>
