<!--This file created 10:48 AM  2/26/98 by Claris Home Page version 2.0-->
<HTML>
<HEAD>
   <TITLE>RECOMB 98</TITLE>
   <META NAME=GENERATOR CONTENT="Claris Home Page 2.0">
   <X-SAS-WINDOW TOP=52 BOTTOM=556 LEFT=8 RIGHT=796>
</HEAD>
<BODY BGCOLOR="#FBF8BC">

<H1><CENTER><B>Recomb 98 Schedule</B></CENTER></H1>

<P><BR>


<H2><B>Saturday March 21, 1998</B></H2>

<P><B><BR>

</B><TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=140>
         <P>7:00 pm - 10:00 pm<BR>
         
         &nbsp;
      </TD><TD VALIGN=top WIDTH=360>
         <P><B>Welcome Reception</B><BR>
         
         Renaissance Room
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H2>Sunday March 22, 1998</H2>

<P><BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=120>
         <P>8:45 am<BR>
         
         &nbsp;
      </TD><TD VALIGN=top WIDTH=580>
         <P><B>Opening Remarks<BR>
         
         </B>Embassy Ballroom<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=120>
         <P><B>Pavel Pevzner<BR>
         
         </B>
      </TD><TD VALIGN=top WIDTH=580>
         <P>RECOMB 1998 Program Committee Chair<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=120>
         <P><B>Gary Benson<BR>
         
         &nbsp;&nbsp;</B>
      </TD><TD VALIGN=top WIDTH=580>
         <P>RECOMB 1998 Conference Chair<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=120>
         <P><B>Sorin Istrail</B><BR>
         
         &nbsp;&nbsp;
      </TD><TD VALIGN=top WIDTH=580>
         <P>RECOMB 1999 Program Committee Chair<BR>
         
         &nbsp;
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top border=0 WIDTH=100>
         <P>9:00 am<BR>
         
      
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Distinguished New Technologies Lecture<BR>
         
         Pavel Pevzner, Chair<BR>
         
         &nbsp;<BR>
         
         David Cox<BR>
         
         The Human Genetic Variation: Oligonucleotide<BR>
         
         Chips and Human Disease</B>
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:00 am
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H3><B>Session 1, Chair: Martin Vingron</B></H3>

<P><BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:10 am<BR>
         
         &nbsp;<BR>
         
         &nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Identifying Satellites in Nucleic Acid
         Sequences</B><BR>
         
         M. Sagot, G. Meyers<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:35 am <B><BR>
         
         &nbsp;<BR>
         
         &nbsp;<BR>
         
         &nbsp;</B>
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Genome Analysis Using Clusters of<BR>
         
         Orthologous Groups (COGs) </B><BR>
         
         E. Koonin, R. Tatusov, M. Y. Galperin, M. N. Rozanov<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:00 am
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H3>Session 2, Chair: Gene Myers</H3>

<P><B><BR>

</B><TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:15 am<BR>
         
         &nbsp;&nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Non-Adjoining Correlations within Signals in
         DNA</B><BR>
         
         P. Agarwal and V. Bafna<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:40 am<BR>
         
         <B>&nbsp;</B>
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>An Algorithm for Finding Tandem Repeats of Unspecified
         Pattern Size<BR>
         
         </B>G. Benson<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>12:05 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Lunch
      </TD></TR>
</TABLE><BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>1:30 pm<BR>
         
         &nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Invited Lecture Session<BR>
         
         Sorin Istrail, Chair<BR>
         
         <BR>
         
         Charles Cantor<BR>
         
         Enhancements in Sequence Analysis with DNA Arrays</B>
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:30 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H3>Session 3, Chair: Gary Benson</H3>

<P><BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:40 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Motif Discovery in Biological Sequences without
         Alignment or Enumeration</B><BR>
         
         I. Rigoutsos, A. Floratos<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:05 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>An Algorithm for Finding Novel Gapped Motifs in DNA
         Sequences</B><BR>
         
         E. Rocke, M. Tompa<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:30 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>A Polyhedral Approach to RNA Sequence Structure
         Alignment</B><BR>
         
         H. Lenhof, K. Reinert, M. Vingron<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:55 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>4:10 pm &nbsp;<BR>
         
         &nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Stanislav Ulam Computational Biology Address</B><BR>
         
         <B>Michael Waterman, Chair&nbsp;<BR>
         
         <BR>
         
         Joshua Lederberg</B>
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=140>
         <P>&nbsp;8:00 pm - 10:00 pm &nbsp;
      </TD><TD VALIGN=top WIDTH=360>
         <P><B>Business Meeting<BR>
         
         </B>Renaissance Room<BR>
         
      
      </TD></TR>
</TABLE></P>

<P><BR>

<BR>

</P>

<H2>Monday March 23, 1998</H2>

<P>Embassy Ballroom<BR>

<BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>&nbsp; 9:00 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Distinguished Conference Lecture<BR>
         
         Chair: Gary Benson<BR>
         
         <BR>
         
         Ron Davis<BR>
         
         Whole Genome Analysis: Expression, Replication,
         Recombination,<BR>
         
         Allelic Variation, and Drug Discovery Function</B>
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:00 am
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H3>Session 4, Chair: David Searls<BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:10 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Assessment of Ab Initio Protein Structure
         Prediction<BR>
         
         </B>A. M. Lesk<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:35 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>A Self-Consistent Field Optimization Approach to<BR>
         
         Build Energetically and Geometrically Correct Lattice<BR>
         
         Models of Proteins<BR>
         
         </B>B. A. Reva, A. V. Finkelstein, J. Skolnick<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:00 am
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break<BR>
         
         &nbsp;<BR>
         
      
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:15 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Modeling DNA Shuffling<BR>
         
         </B>F. Sun<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:40 am<BR>
         
      
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Multiple Genome Rearrangements<BR>
         
         </B>D. Sankoff, M. Blanchette<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>12:05 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Lunch
      </TD></TR>
</TABLE><BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>1:30 pm &nbsp;<BR>
         
         &nbsp;&nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Invited Lecture Session<BR>
         
         Chair: Eugene Koonin<BR>
         
         <BR>
         
         Ruben Abagyan<BR>
         
         Challenges of Biomolecular Structure Prediction</B>
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:30 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

<BR>

<BR>

Session 5, Chair: Martin Farach</H3>

<P><BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:40 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>From Four-taxon Trees to Phylogenies: The Case of<BR>
         
         Mammalian Evolution<BR>
         
         </B>A. Ben-Dor, B. Chor, D. Graur, R. Ophir, D. Pelleg<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:05 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>On Reconstructing Species Trees From Gene Trees In
         Term<BR>
         
         of Duplications<BR>
         
         </B>B. Ma, M. Li, L. Zhang<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:30 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>The Ordinal Quartet Method<BR>
         
         </B>P. E. Kearney<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:55 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break</P>
         
         <P>&nbsp;
      </TD></TR>
</TABLE><BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>4:10 pm &nbsp;<BR>
         
         &nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Invited Lecture Session<BR>
         
         Bonnie Berger, Chair<BR>
         
         <BR>
         
         Michael Levitt<BR>
         
         Bridging the Gap between Sequence and Structure</B>
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=140>
         <P>6:00 pm-10:00 pm
      </TD><TD VALIGN=top WIDTH=360>
         <P><B>Poster Session<BR>
         
         </B>Renaissance Room</P>
         
         <P>&nbsp;
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H2>Tuesday March 24, 1998</H2>

<P>Embassy Ballroom<BR>

<BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>9:00 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Invited Lecture Session<BR>
         
         Ron Shamir, Chair<BR>
         
         <BR>
         
         David Schwartz&nbsp;<BR>
         
         New Approaches to Genomic Analaysis Using Single
         Molecules</B>
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:00 am
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H3>Session 6, Chair: Minoru Kanehisa</H3>

<P><BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:10 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Algorithms for Optical Mapping<BR>
         
         </B>R. M. Karp, R. Shamir<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:35 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Estimation for Restriction Sites Observed by
         Optical<BR>
         
         Mapping Using Reversible-jump Markov Chain Monte Carlo<BR>
         
         </B>J. K. Lee, V. Dancik, M. S. Waterman<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:00 am
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break<BR>
         
         &nbsp;<BR>
         
      
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:15 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Partitioning K Clones: Hardness Results and a
         Practical<BR>
         
         Solution to the K-Popultaion Problem<BR>
         
         </B>L. Parida, B. Mishra<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:40 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Constructing Maps Using the Span and Inclusion
         Relations<BR>
         
         </B>D. Fasulo, T. Jiang, R. M. Karp, N. Sharma<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>12:05 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Lunch
      </TD></TR>
</TABLE><BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>1:30 pm &nbsp;<BR>
         
         &nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Invited Lecture Session<BR>
         
         Chair: Webb Miller<BR>
         
         <BR>
         
         John Yates&nbsp;<BR>
         
         Mass Spectrometry and the Proteome</B>
      </TD></TR>
</TABLE><BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:30 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H3>Session 7, Chair: Thomas Lengauer<BR>

</H3>

<P><BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:40 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>How Fast a Protein Chain Can Fold to Its Most Stable
         Structure?<BR>
         
         </B>A. V. Finkelstein, A. Ya. Badretdinov<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:00 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Algorithmic Determination of Core Positions in the VL
         and<BR>
         
         VH Domains of Immunoglobulin Molecules</B><BR>
         
         I. Gelfand, A. Kister, C. Kulikowski, O. Stoyanov<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:30 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Alignments Without Low-Scoring Regions</B><BR>
         
         Z. Zhang, P. Berman, W. Miller<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:55 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Dynamic Programming Alignment Accuracy</B><BR>
         
         I. Holmes, R. Durbin<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD WIDTH=100>
         <P>4:20 pm
      </TD><TD WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

<TABLE BORDER=1 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>4:30 pm</P>
         
         <P>&nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Invited Lecture Session<BR>
         
         Chair: David Haussler<BR>
         
         <BR>
         
         Klaus Gubernator<BR>
         
         Evolutionary Drug Discovery</B>
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H2>Wednesday March 25, 1998</H2>

<P>Embassy Ballroom<BR>

</P>

<P><BR>

<BR>

</P>

<H3>Session 8, Chair: Dan Gusfield<BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>8:30 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Protein Folding in the Hydrophobic-Hydrophilic (HP)
         Model is NP-Complete</B><BR>
         
         B. Berger, T. Leight<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>8:55 am</P>
         
         <P>&nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>On the Complexity of Protein Folding</B><BR>
         
         P. Crescenzi, D. Goldman, C. Papadimitriou, A. Piccolboni,
         M.Yannakakis<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>9:20 am</P>
         
         <P>&nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>A New Method for Modeling and Solving the Protein Fold
         Recognition Problem</B><BR>
         
         Y. Xu, D. Xu, E. C. Uberbacher<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>&nbsp;9:45 am</P>
         
         <P>&nbsp;
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Optimal Detection of Sequence Similarity by Local
         Alignment</B><BR>
         
         T. Hwa, M. Lassig<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:10 am
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break<BR>
         
      
      </TD></TR>
</TABLE><BR>

<BR>

<BR>

Session 9, Chair: Terry Speed</H3>

<P><BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:20 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>A Formally Exact Method to Numerically Analyze Local
         Denaturation<BR>
         
         in Superhelical DNA</B><BR>
         
         R. M. Fye, C. J. Benham<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>10:45 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>The Theoretical Limits of DNA Sequence Discrimination
         of Polyamides</B><BR>
         
         W. L. Walker, D. S. Goodsell, E. M. Landaw<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:10 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Maxwell Demon and Topology Simplification by Type II
         Topoisomerases</B><BR>
         
         A. Vologodskii<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>11:35 am
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>A Unified Approach to Word Statistics</B><BR>
         
         M. Regnier<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>12:00 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Lunch&nbsp;
      </TD></TR>
</TABLE><BR>

</P>

<P><BR>

<BR>

</P>

<H3>Session 10, Chair: Phil Green<BR>

<BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>1:00 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Regression Analysis of Multiple Protein
         Structures</B>&nbsp;<BR>
         
         T. D. Wu, S. C. Schmidler, T. Hastie, D. L. Brutlag<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>1:25 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Better Methods for Solving Parsimony and
         Compatiblity</B><BR>
         
         M. Bonet, M. Steel, T. Warnow, S. Yooseph<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>1:50 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Beyond Mutation Matrices: Physical-Chemistry Based<BR>
         
         Evolutionary Models</B><BR>
         
         J. M. Koshi, D. P. Mindell, R. A. Goldstein<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:15 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>A Structure Based Similarity Measure for Nucleic
         Acid<BR>
         
         Sequence Comparison</B><BR>
         
         R. Liu, T. W. Blackwell and D. J. States<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:40 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P>Break
      </TD></TR>
</TABLE><BR>

<BR>

<BR>

Session 11, Chair: David Benson</H3>

<P><BR>

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=500>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>2:55 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>The Hierarchical Organization of Molecular Structure
         Computations</B><BR>
         
         C. C. Chen, J. P. Singh, R. B. Altman<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:20 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Estimaiton of Allele Frequencies From
         Color-Multiplexed<BR>
         
         Electropherograms</B><BR>
         
         D. G. Politte, D. R. Maffitt, D. J. States<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>3:45 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Analysis of the Position Dependent Amino Acid
         Probabilities<BR>
         
         and its Applicationts to the Search for Remote
         Homologues</B><BR>
         
         S. R. Sunyaev, I. V. Rodchenkov, F. Eisenhaber, E. N.
         Kuznetsov<BR>
         
         &nbsp;
      </TD></TR>
   <TR>
      <TD VALIGN=top WIDTH=100>
         <P>4:10 pm
      </TD><TD VALIGN=top WIDTH=400>
         <P><B>Family-based Homology Detection via Pairwise Sequence
         Comparison</B><BR>
         
         W. N. Grundy<BR>
         
         &nbsp;
      </TD></TR>
</TABLE><BR>

</P>

<BLOCKQUOTE><BLOCKQUOTE><P><B>End of RECOMB 98<BR>

</B></P></BLOCKQUOTE></BLOCKQUOTE>

<P>&nbsp;</P>

<P>
<HR>
</P>

<H5><I><A HREF="https://recomb-conference.github.io/recomb1997">Link to RECOMB
97</A></I></H5>

<P>&nbsp;</P>

</BODY>
</HTML>
